core/src/test/scala/com/raphtory/api/analysis/graphstate/AccumulatorTest.scala

```diff
@@ -92,7 +91,7 @@ class AccumulatorTest extends BaseCorrectnessTest {
       ) // this makes prometheus start on a random unused port
       .use { graph =>
         IO {
-          graph.load(Source(ResourceSpout("MotifCount/motiftest.csv"), BasicGraphBuilder))
+          graph.load(setSource())
           val job = graph
```

Comment:
This test is unnecessarily messy (not your fault). This should really use `graphS` instead of creating a new graph here which gives access to the suite-wide graph fixture.